seedlet-blocks/child-theme.json

```diff
@@ -75,9 +75,9 @@
 					"name": "Background"
 				},
 				{
-					"slug": "selection",
+					"slug": "tertiary",
 					"color": "#f2f2f2",
```

Comment:
Please double check these colors against the original Seedlet theme. [The value of `tertiary` there is different](https://github.com/Automattic/themes/blob/1aa5058bcff44951877c2db4f461fd4a46433613/seedlet/style.css#L112), and should replace this. I'm not sure where this color value came from.

Reply:
double-checked and changed